kratos/containers/container_variable_data/expressions/binary/binary_expression.h

```diff
@@ -0,0 +1,87 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#pragma once
+
+// System includes
+#include <cmath>
+#include <string>
+#include <vector>
+
+// Project includes
+#include "containers/container_variable_data/expressions/expression.h"
+
+namespace Kratos {
+
+///@name Kratos Classes
+///@{
+
+namespace BinaryOperations
+{
+    struct Addition       { static inline constexpr double Evaluate(const double V1, const double V2) { return V1 + V2; } };
+    struct Substraction   { static inline constexpr double Evaluate(const double V1, const double V2) { return V1 - V2; } };
+    struct Multiplication { static inline constexpr double Evaluate(const double V1, const double V2) { return V1 * V2; } };
+    struct Division       { static inline constexpr double Evaluate(const double V1, const double V2) { return V1 / V2; } };
+    struct Power          { static inline constexpr double Evaluate(const double V1, const double V2) { return std::pow(V1, V2); } };
+}
```

Comment:
I'd put these structs into the `BinaryExpression` class def instead of the `Kratos::BinaryOperations` namespace. It'd be a bit more clear what they're supposed to be used for when someone is browsing this header but doesn't know anything about the system.

Reply:
I;m not sure this is possible since the `BinaryExpression` class is templated, and these template arguments are `BinaryOperations`. How is it possible to put the template arguments within the templated class and instantiate in this case? :D.... :/

Reply:
ah yes, I see now. Nevermind then, it's fine as it is.